index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/439f1d2750.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Trombones</title>
+  </head>
+  <body>
+    <div class="Center">
+      <header id="header">
+        <div id="logo">
+          <img
+            id="Trumpet"
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+            alt="original trombones logo"
+          />
+        </div>
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a href="#features">Features</a></li>
+            <li><a href="#how-it-works">How it works</a></li>
+            <li><a href="#pricing">pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <section class="form">
+        <h1>Handcrafed, home-made masterpieces</h1>
+        <form action="https://www.freecodecamp.com/email-submit        ">
+          <input
+            type="email"
```

Comment:
[OPTIONAL] You already have a header and a footer, it should be better for best practice to put this part of project in a <main> tag 😃 👍